core/constants.py

```diff
@@ -67,7 +67,7 @@ def remove_comments(text: str) -> str:
 # need utils.get_package_file_contents but it does not have it loaded to memory
 # yet. If called from utils we get error as `module has no attribute`.
 def get_package_file_contents(
-    package: str, filepath: str, mode: str = 'r'
+    package: str, filepath: str, is_binary: bool = False
```

Comment:
Rather use `binary_mode` instead of `is_binary`

Reply:
Updated